packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md

```diff
@@ -1,12 +1,15 @@
 ---
 title: Customization examples
 ---
-Use the following examples to help you customize your sign-in page with your own CSS, scripts, and per-application branding.
+
+Use the following examples to help you customize the sign-in page with your own CSS, scripts, and per-application branding.
 
 ## Add your own stylesheet
```

Comment:
I think this subheading, and the others in this section, should be smaller, maybe h3's, to make it clear they're sub sections of "Customization examples".